advocacy_docs/partner_docs/QuestFoglightforDatabases/05-UsingQuestFoglightforDatabases.mdx

```diff
@@ -0,0 +1,74 @@
+---
+title: 'Using Quest Foglight for Databases to Monitor Health of EDB Postgres Advanced Server and/or EDB Postgres Extended Server Databases'
```

Comment:
This title is too long, let's just make sure both the page title and file are the same. Change it to "Using Quest Foglight for Databases"